### PR TITLE
Domains: Add enable/disable privacy for registered domains

### PR DESCRIPTION
**Pull request by aidvu**
#### Changes proposed in this Pull Request

We made privacy free, so now we want to simplify the logic
and allow users to just be able to toggle privacy instead of
having to go through checkout, or to cancel the privacy sub
every time.

Depends on D24443-code.

#### Testing instructions

- Open a registered domain
- Go to privacy screen
- Toggle privacy on-off
- Make sure all is working correctly
- Confirm the removed screen is used nowhere else

### CODE REVIEW

File: client/my-sites/domains/domain-management/contacts-privacy/card.jsx [hunk 1]
```diff
@@ -110,29 +137,49 @@ class ContactsPrivacyCard extends React.PureComponent {
 			);
 		}
 
+		let privacyText = translate(
+			'{{strong}}Privacy Protection{{/strong}} is turned {{strong}}off{{/strong}} for this domain. ' +
```
Comment (a8ci18n) [inline client/my-sites/domains/domain-management/contacts-privacy/card.jsx]: Hi! I've found a possible matching string that has already been translated **21** times:
`translate( '{{strong}}Privacy Protection{{/strong}} is turned off for this domain. Your contact information is {{strong}}public{{/strong}}. {{a}}Enable Privacy Protection{{/a}}' )` *ES Score: 19*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/domains/domain-management/contacts-privacy/card.jsx [hunk 2]
```diff
@@ -110,29 +137,49 @@ class ContactsPrivacyCard extends React.PureComponent {
...
+			{
+				components: {
+					strong: <strong />,
+				},
+			}
+		);
+
+		if ( privateDomain ) {
+			privacyText = translate(
+				'{{strong}}Privacy Protection{{/strong}} is turned {{strong}}on{{/strong}} for this domain. ' +
```
Comment (a8ci18n) [inline client/my-sites/domains/domain-management/contacts-privacy/card.jsx]: Hi! I've found a possible matching string that has already been translated **21** times:
`translate( '{{strong}}Privacy Protection{{/strong}} is turned off for this domain. Your contact information is {{strong}}public{{/strong}}. {{a}}Enable Privacy Protection{{/a}}' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/domains/domain-management/contacts-privacy/card.jsx [hunk 3]
```diff
@@ -110,29 +137,49 @@ class ContactsPrivacyCard extends React.PureComponent {
...
-			</Notice>
+			<div className="contacts-privacy__info">
+				<p>{ privacyText }</p>
+				<Button
+					className="contacts-privacy__info-button"
+					disabled={ submitting }
+					busy={ submitting }
+					onClick={ this.togglePrivacy }
+				>
+					{ privateDomain ? translate( 'Disable Privacy' ) : translate( 'Enable Privacy' ) }
```
Comment (a8ci18n) [inline client/my-sites/domains/domain-management/contacts-privacy/card.jsx]: Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Disable Whois privacy.' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by aidvu** [comment]
> Just a suggestion, feel free to ignore - it's not like the rest of the Domain Management is any better. Or ping Dan for input :)

You know my favorite word... Iterate. And also, pinging @danhauk.

**Pull request merged**